src/resolvers/queries/notablePerson.ts

```diff
@@ -0,0 +1,53 @@
+import { connection } from '../../database/connection';
+import { NotablePerson } from '../../database/entities/notablePerson';
+import { NotablePersonEvent } from '../../database/entities/event';
+import { NotablePersonEventComment } from '../../database/entities/comment';
+import { NotablePersonRootQueryArgs } from '../../typings/schema';
+
+export const notablePersonResolvers = {
+  RootQuery: {
+    async notablePerson(_: undefined, { slug }: NotablePersonRootQueryArgs) {
```

Comment:
Where does `NotablePersonRootQueryArgs` come from?

Reply:
TypeScript types are generated from the `schema.graphql` by running this command:

```
yarn generate-schema-types
```

(this is explained in the wiki entry)